src/projections/placeholder.js

```diff
@@ -0,0 +1,18 @@
+import { compact, last, pluck, assign } from '../utils';
+
+export default function ({
+  composeSections,
+}) {
+  return {
+    composeSections(section) {
+      const { tag, table, trs = [] } = section;
+      if (tag === 'TBODY' && trs.length === 0) {
+        const placeholders = pluck([table, section], 'placeholder');
```

Comment:
We should be able to config `placeholder` on `tr`. In case the `tr` has an `[]` as `data`, show the placeholder.